### DISCUSSION TITLE
Musical suppression and audio restoration

### DISCUSSION THREAD

**Discussion by Brodvd** [intel/openvino-plugins-ai-audacity#214] [Ideas]
Hello everyone, I thank the developers for their effort for openVINO AI plugins, but I would like to propose two ideas about new IA plugins: 1. Noise suppression per voice is quite good, you just need to improve it in detail, but honestly it is also necessary a noise suppression for musical instruments, VST or recordings, and consequently to expand musical separation as they mention in so many. 2. Instead I recently found myself in a recording that presented a considerable level of clipping, I know that the best thing is to avoid it but I could not record again, so I used the "clip correction" of audacity but I didn’t get much, so I was thinking if it wasn’t possible to create an AI plugin for rebuilding distorted samples, and seeing other discussions, it might be a good idea to make a plugin that improves and restores damaged audio, like a "super resolution" in photo editing.

**Comment by MarcoRavich** [comment]
I've collected some for [HyMPS project](https://www.forart.it/HyMPS/) in [AUDIO section](https://github.com/FORARTfe/HyMPS#-) \ [AI-based page](https://github.com/FORARTfe/HyMPS/blob/main/Audio/AI-based.md#--), enjoy.

**Reply by Brodvd** [reply]
Great  @MarcoRavich  They seem to me to be very good models.

**Comment by Brodvd** [comment]
@RyanMetcalfeInt8 since many on this reportery ask to have more stems in the music separation plugin, these models of @MarcoRavich could work and could add with openVINO?

**Reply by MarcoRavich** [reply]
> @RyanMetcalfeInt8 since many on this reportery ask to have more stems in the music separation plugin, these models of @MarcoRavich could work and could add with openVINO?
>
Asked [here](https://github.com/intel/openvino-plugins-ai-audacity/discussions/77) already.
